src/consensus/aft/raft.h

```diff
@@ -436,14 +436,25 @@ namespace aft
     std::pair<Term, Index> get_commit_term_and_idx()
     {
       std::lock_guard<std::mutex> guard(state->lock);
-      return {get_term_internal(state->commit_idx), state->commit_idx};
+      if (consensus_type == ConsensusType::CFT)
+      {
+        return {get_term_internal(state->commit_idx), state->commit_idx};
+      }
+      else
+      {
+        auto progress_tracker = store->get_progress_tracker();
+        auto idx = progress_tracker->get_rollback_seqno();
```

Comment:
What's the rollback seqno, is the index of the last committed transaction?

Reply:
This is the latest prepared which is >= committed